### PR TITLE
Add descriptions about getPeerConnection()

### PR DESCRIPTION
**Pull request by monmee**
refs: https://github.com/skyway/skyway-js-sdk/pull/179

### CODE REVIEW

File: docs/ja/mediaconnection.md [hunk 1]
```diff
@@ -98,6 +98,25 @@ peer.on('call', mediaConnection => {
...
 
+### `getPeerConnection()`
+
+!!! このメソッドは上級者ユーザ向け機能です。"
+
+接続先のPeer間とのWebRTCコネクションである `RTCPeerConnection` を取得します。
+
+#### Return value
+
+A [RTCPeerConnection](https://developer.mozilla.org/ja/docs/Web/API/RTCPeerConnection) instance
```
Comment (leader22) [inline docs/ja/mediaconnection.md]: We prefer to use `link` notation.

See actual link of `MediaStream` in this file.

Comment (monmee) [reply docs/ja/mediaconnection.md]: Thank you for your review. We fixed it.

File: docs/ja/mediaconnection.md [hunk 2]
```diff
@@ -98,6 +98,26 @@ peer.on('call', mediaConnection => {
 
 `undefined`
 
+### `getPeerConnection()`
+
+接続先のPeer間とのWebRTCコネクションである `RTCPeerConnection` を取得します。
+
+!!! 注意
+  取得したRTCPeerConnectionを誤ってカスタマイズした場合、SDKが正しく動作しなくなる可能性があります。
```
Comment (leader22) [inline docs/ja/mediaconnection.md]: `誤ってカスタマイズ`...? 🤔

Comment (monmee) [reply docs/ja/mediaconnection.md]: We fixed sentences:D

File: docs/en/mediaconnection.md [hunk 3]
```diff
@@ -101,6 +101,26 @@ and receive mode even if the former connection was in receive only mode.
 
 ## Events
 
+### `getPeerConnection()`
+
+Get `RTCPeerConnection` which is WebRTC connection between remote peer.
+
+!!! Notice
+  If RTCPeerConnection is handled by mistake、there is a risk that SDK does not work.
```
Comment (matsub) [inline docs/en/mediaconnection.md]: `handled by mistake` sounds odd to me. How about

```suggestion
Note that the side effects of use of RTCPeerConnection can make a risk that SkyWay SDK to behave unexpectedly.
```

File: docs/en/mediaconnection.md [hunk 4]
```diff
@@ -101,6 +101,26 @@ and receive mode even if the former connection was in receive only mode.
...
+
+[RTCPeerConnection] instance
+
+#### Sample
+
+```js
+const mc = peer.call('peerID', mediaStream);
+const pc = mc.getPeerConnection();
+// ...
+});
```
[the remark this answers is not in the document]
Comment (matsub) [reply docs/en/mediaconnection.md]: i mean line `121`

File: docs/en/mediaconnection.md [hunk 5]
```diff
@@ -101,6 +101,26 @@ and receive mode even if the former connection was in receive only mode.
 
 ## Events
 
+### `getPeerConnection()`
+
+Get `RTCPeerConnection` which is WebRTC connection between remote peer.
```
[the remark this answers is not in the document]
Comment (iwashi) [reply docs/en/mediaconnection.md]: > An RTCPeerConnection instance allows an application to establish peer-to-peer communications with another RTCPeerConnection instance in another browser, or to another endpoint implementing the required protocols. Communications are coordinated by the exchange of control messages (called a signaling protocol) over a signaling channel which is provided by unspecified means, but generally by a script in the page via the server, e.g. using XMLHttpRequest [XMLHttpRequest] or Web Sockets [WEBSOCKETS-API].

is from https://www.w3.org/TR/webrtc/

and 

> The RTCPeerConnection interface represents a WebRTC connection between the local computer and a remote peer. It provides methods to connect to a remote peer, maintain and monitor the connection, and close the connection once it's no longer needed.

is from https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection .

**Comment by leader22** [comment]
@matsub Would you please review this again...?

**Pull request merged**